src/python/dxpy/dx_extract_utils/ExpressionInputsValidator.py

```diff
@@ -0,0 +1,164 @@
+from ..exceptions import err_exit
+
+
+class ExpressionInputsValidator:
+    """InputsValidator class for extract_assay expresion. Checks for invalid input combinations"""
+
+    def __init__(self, args, error_handler):
```

Comment:
Let's have a default value for `error_handler` so that it can print something if none is provided. I recommend Python's own `print` function which will work in both Python 2 and 3

```suggestion
    def __init__(self, args, error_handler=print):
```